_pages/en_US/include/soundhaxploit.txt

```diff
@@ -0,0 +1,62 @@
+### What You Need
```

Comment:
the include should probably be called soundhax-exploit or soundhaxexploit instead of soundhaxploit

Reply:
or why not just straight "soundhax" since that's what the include does

Reply:
mhm